README.rst

```diff
@@ -43,7 +43,7 @@ You can install ``MXNet`` and ``GluonNLP`` using pip:
 
 ::
 
-    pip install --pre --upgrade mxnet
```

Comment:
Instead of maintaining a separate page, would adding a reference to the installation guide be easier?

Also, the version of mxnet to install may depend on the purpose. For example, we may want to use nightly for development on master, while using official releases for use with released versions of gluonnlp.

Reply:
Good point, updated the page accordlingly.